package-lock.json

```diff
@@ -0,0 +1,4883 @@
+{
```

Comment:
Can you delete this file because we use pnpm?

Reply:
ok ok